src/components/expired-subscription-modal/tests/ExpiredSubscriptionModal.test.jsx

```diff
@@ -19,13 +19,13 @@ const mockEnterpriseCustomer = enterpriseCustomerFactory();
 const defaultAppContextValue = { authenticatedUser: mockAuthenticatedUser };
 const ExpiredSubscriptionModalWrapper = ({ children, appContextValue = defaultAppContextValue }) => (
```

Comment:
nit: should this wrapper component's name be updated to reflect the changed component name?